filament/include/filament/Skybox.h

```diff
@@ -145,6 +156,8 @@ class UTILS_PUBLIC Skybox : public FilamentAPI {
 
     void setColor(math::float4 color) noexcept;
 
+    void setType(SkyboxType type) noexcept;
```

Comment:
The only case, when it could cause trouble, is if the environment is not set (making the cubemap texture `nullptr`). I checked in gltf viewer, Filament does neither crash nor complain, and simply makes the background black.